bash/hail

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Why not bin/?  bash/ seems like a strange choice.  For build, in general I think of two options: (1) the build should populate the installation in the root directory (but there will be extra stuff from original source files and intermediate build objects) or (2) you build into a pristine location.  Are we (or do we want to be) in the first or second regime?